### PR TITLE
Update changelog for 3.0.8

### PR DESCRIPTION
**Pull request by blink1073**
## References
Updates changelog for upcoming 3.0.8

## Code changes
None

## User-facing changes
Updated docs

## Backwards-incompatible changes
None

### CODE REVIEW

File: docs/source/getting_started/changelog.rst [hunk 1]
```diff
@@ -10,6 +10,15 @@ See the `JupyterLab
...
 milestone on GitHub for the full list of pull requests and issues closed.
 
+v3.0.8
+^^^^^^
+* Add hash to webpack requests to enable caching.  (`#9776 <https://github.com/jupyterlab/jupyterlab/pull/9776>`__)
+* Update MANIFEST.in to include package_data files.  (`#9780 <https://github.com/jupyterlab/jupyterlab/pull/9780>`__)
+* Correct synchronization of tags between metadata and tags widget.  (`#9773 <https://github.com/jupyterlab/jupyterlab/pull/9773>`__)
+* Fix use of hyphen in module name.  (`#9655 <https://github.com/jupyterlab/jupyterlab/pull/9655>`__)
+* Add missing default_url fields to more examples.  (`#9737 <https://github.com/jupyterlab/jupyterlab/pull/9737>`__)
+* Add missing default_url fields to examples.  (`#9731 <https://github.com/jupyterlab/jupyterlab/pull/9731>`__)
```
Comment (jasongrout) [inline docs/source/getting_started/changelog.rst:20]:
```suggestion
* Add missing default_url fields to examples.  (`#9731 <https://github.com/jupyterlab/jupyterlab/pull/9731>`__, `#9737 <https://github.com/jupyterlab/jupyterlab/pull/9737>`__)
```

**Review by jasongrout — APPROVED** [review]
Looks good, with one suggestion above.

**Comment by blink1073** [comment]
Releasing now, if my power stays on...

**Comment by blink1073** [comment]
@meeseeksdev please backport to 3.0.x

**Pull request merged**